db/migrate/20400904110441_remove_unique_index_from_identification.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveUniqueIndexFromIdentification < ActiveRecord::Migration
+
+  def change
+    remove_index :logins, :identification
```

Comment:
This is an interesting one. I think given it's confusing appearance, maybe we should remove the first index by name: http://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/remove_index

That way, it's a little more obvious what's happening (obviously, this is already in the name of the class, too)